test/route-install.js

```diff
@@ -55,6 +63,10 @@ describe('install route', function () {
           .get('/pkg')
           .times(2)
           .reply(200, this.pkgData);
+
+      nock('http://yeoman.io')
+        .get('/blacklist.json')
+          .reply(200, this.blacklist);
```

Comment:
@kevva lines above are indented the same way, since that is the reply for that specific get request.
```
   nock(registryUrl)
        .get('/' + npmKeyword.url('yeoman-generator').replace(registryUrl, ''))
          .reply(200, { rows: this.rows })   
```